src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -80,6 +80,12 @@ public function reset(): void
         $this->doMysqlCleanup = false;
     }
 
+    public function close(): void
+    {
+        $this->driverConnection->close();
```

Comment:
this transport does not own the DBAL connection, so it should not close it either

Reply:
Updated. Other transports seem OK to me to be closeable (they remain their own connection or already have some "reset" methods).

Reply:
@stof from a messenger:consume POV i think doctrine transport is "closeable"

that's what DoctrineCloseConnectionMiddleware does no?

Reply:
DoctrineCloseConnectionMiddleware is closing the connection *if you enable that middleware*.

the transport itself should not close the connection implicitly, because it is not owning the connection (the connection might be used for multiple purposes)